### ISSUE TITLE
Container Registry 'unknown' is not a valid ArtifactArchitecture

### ISSUE THREAD

**Issue by dmartin-r7** [Azure/azure-sdk-for-python#30077]
- **Package Name**: azure-containerregistry
- **Package Version**: 1.0.0
- **Operating System**: macOS
- **Python Version**: 3.8

**Describe the bug**
Container Registry list call throws the error below.

**To Reproduce**
Call `container_registries.list()`

```
ValueError: 'unknown' is not a valid ArtifactArchitecture
(5 additional frame(s) were not displayed)
...
  File "/usr/local/lib/python3.8/dist-packages/azure/containerregistry/_container_registry_client.py", line 243, in <listcomp>
    ArtifactManifestProperties._from_generated(  # pylint: disable=protected-access
  File "/usr/local/lib/python3.8/dist-packages/azure/containerregistry/_models.py", line 67, in _from_generated
    return cls(
  File "/usr/local/lib/python3.8/dist-packages/azure/containerregistry/_models.py", line 48, in __init__
    self._architecture = ArtifactArchitecture(self._architecture)
  File "enum.py", line 339, in __call__
    return cls.__new__(cls, value)
  File "enum.py", line 663, in __new__
    raise ve_exc
```

Kwargs being passed in to this model init:

```
{
    can_delete: True, 
    can_read: True, 
    can_write: True, 
    cpu_architecture: 'unknown', <----------
    created_on: datetime.datetime(2023, 1, 31, 13, 50, 13, 218393, tzinfo=<isodate.tzinfo.Utc object at 0x7f6a96f7a760>), 
    digest: -removed-, 
    last_updated_on: datetime.datetime(2023, 1, 31, 13, 50, 13, 218393, tzinfo=<isodate.tzinfo.Utc object at 0x7f6a96f7a760>), 
    operating_system: 'unknown', 
    size_in_bytes: 1263, 
    tags: None
}
```

**Expected behavior**
Error should not occur.

**Comment by kashifkhan** [comment]
Thank you for the feedback @dmartin-r7. We will investigate and get back to you asap.

**Comment by YalinLi0312** [comment]
Hi @dmartin-r7 , thanks for reaching us! We have met the issue and working on it now, it will be fixed in our next release.

**Comment by YalinLi0312** [comment]
Hi @dmartin-r7 , can you try it with our [latest preview](https://pypi.org/project/azure-containerregistry/1.1.0b4/)? We have some updates in deserialization.

**Comment by YalinLi0312** [comment]
Got confirmed from other users that the issue is still exist. We are checking with service team about the details.

**Comment by dmartin-r7** [comment]
Hi still seeing the same as Yalin confirmed above.

**Comment by YalinLi0312** [comment]
Hi @dmartin-r7 - We published a release ([version 1.2.0](https://pypi.org/project/azure-containerregistry/)) to fix this issue today, please give it a try. Feel free to let us know if there still has any issue. Thanks for your patience!
